pkgs/tools/security/otpgen/default.nix

```diff
@@ -0,0 +1,48 @@
+{ 
+  pkgs
+  , lib
+  , stdenv
+  , makeWrapper
+  , fetchFromGitHub
+  , cmake
+  , pkg-config
+  , cryptopp
+  , qtbase
+}:
+
+stdenv.mkDerivation rec {
+
+  pname = "OTPGen";
+  version = "0.9.3";
+
+  src = fetchFromGitHub {
+    owner = "magiruuvelvet";
+    repo = "OTPGen";
+    rev = "v${version}";
+    sha256 = "19nyr3gmfxak0zdzibs1pjcdaxyy8x7jvrhgi3bsg9jbgf1m6xsr";
+  };
+
+  patches = [ 
+
+    # Replacing libcrypto++ dependency with libcryptopp. 
+    # Patch on discussion https://github.com/magiruuvelvet/OTPGen/pull/3
+    ./core-cmake-libcrypto++.patch 
+
+    # Removing unrecognized -rpath option.
+    # Patch on discussion https://github.com/magiruuvelvet/OTPGen/pull/2
+    ./cmakelists-rpath.patch 
+  
```

Comment:
Space here is not necessary.

Reply:
Fixed here https://github.com/NixOS/nixpkgs/pull/87887/commits/f70af1e46c6f7fa18556a83889bde4547972551a